### ISSUE TITLE
Error 404

### ISSUE THREAD

**Issue by banana-club-cloud** [RocketChat/Rocket.Chat.Livechat#358]
Hi,

when I am using the link to the livechat from another server, 
or if I am using the iframe script, I get a 404 error.
From the same server, links and iframes work.

Note: I enabled iframes for all domains in the settings for testing purposes.
Later I want to restrict this just to my domains.
I would be very happy, if someone could help me.

Thank you

**Comment by Brandon1811** [comment]
hi @banana-club-cloud I believe I am also experiencing this issue, the livechat widget does not work inside the iframe or in any window opened via the JS api on server B & throws a 404 page not found error.  /livechat works if i enter the URL into a new tab manually (http request referer is not set to server B). were you ever able to resolve?

I also asked this question/seeking guidance on the support & omnichannel-dev channels of open.rocket.chat but haven't heard back as of yet ( https://open.rocket.chat/channel/support & https://open.rocket.chat/channel/omnichannel-dev )

If anyone has any ideas/suggestions please reach out it would be much appreciated!
Thanks

**Comment by renatobecker** [comment]
Is there any error on the browser console?
`Network` -> `XHR`

**Comment by Brandon1811** [comment]
Hi @renatobecker - I posted this to you on the omnichannel dev channel (https://open.rocket.chat/channel/omnichannel-dev ) but will follow up here for completeness. 
1) no network errors anywhere (xhr or otherwise)

i wiped my previous installations (snap & a manual installation both on ubuntu) and installed via docker-compose & all seems to be working.

I did experience the 404 error again when i attempted to lock my omnichannel install down by adding my domain to the Livechat Allowed Domains setting. It didn't seem to work w/o a www prefix. ie. www.{my-domain}.com worked, while {my-domain}.com resulted in the 404 page in the livechat widget. unfortunately i can't be 100% if installing via docker fixed or if it was because of my Livechat Allowed Domains setting as I can't know for sure what that setting was in the previous installs - had been tweaking things trying to get working - but i can confirm what I'm seeing now - leaving that setting blank and livechat works (expected), setting to www.{my-domain}.com works (expected), but {my-domain}.com does not work / throws 404

Maybe if anybody else encounters the same thing this experience could be of some use. 

Long story short - All good now w/ a docker-compose install + Livechat Allowed Domains setting of either empty or www.{my-domain}.com. 

Thank you!
